src/components/custom/useMovieContext.js

```diff
@@ -0,0 +1,8 @@
+import React, { useContext } from 'react';
+import { MovieContext } from '../../App';
+
+export const useMovieContext = () => {
+  const [movieDetail, setMovieDetail] = useContext(MovieContext);
```

Comment:
Not critical: normally you want to use an object here because the more props you return the more difficult it gets to handle them based on the order.

Reply:
Noted